xlml/utils/metric.py

```diff
@@ -406,7 +409,7 @@ def get_gke_job_status(benchmark_id: str) -> bigquery.JobStatus:
 
 
 def get_gce_job_status(
-    task_test_config: test_config.TestConfig[test_config.Tpu], use_startup_script: bool
+    task_test_config: test_config.TestConfig[T], use_startup_script: bool
```

Comment:
Will you meet issue when you apply `test_config.TestConfig[Accelerator]`?

Reply:
Nice catch, I forgot we have `Accelerator` here. Will do a try once the current run is complete.